contrib/reports/explore-server/src/client/MyCodeMirrors.tsx

```diff
@@ -4,8 +4,7 @@ import * as ReactCodeMirror from "react-codemirror2";
 import * as ReactDOM from "react-dom";
 import * as Conclusions from "./Conclusions";
 import { Alert, Weakness } from "../shared/shared-types";
-import { getToolIcon, getToolOrderID } from "./util";
-import { rangeToArray } from "../../../../../src/util";
+import { getToolIcon, getToolOrderID, rangeToArray } from "./util";
```

Comment:
also, I've made an import mistake here which was preventing webpack from running, it's fixed now